dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ExecutorServiceImpl.java

```diff
@@ -724,8 +724,7 @@ private int createCommand(CommandType commandType, long processDefineCode, TaskD
                 return createComplementCommandList(schedule, runMode, command, expectedParallelismNumber,
                     complementDependentMode);
             } catch (CronParseException cronParseException) {
-                // this just make compile happy, since we already validate the cron before
-                logger.error("Parse cron error", cronParseException);
+                // We catch the exception here just to make compiler happen, since we have already validated the schedule cron expression before
```

Comment:
```suggestion
                // We catch the exception here just to make compiler happy, since we have already validated the schedule cron expression before
```